### ISSUE TITLE
Example for climate entities 

### ISSUE THREAD

**Issue by soyxan** [fmtr/hct#2]
Hi,

I already integrated the internal Tasmota theremostat in HA with a custom berry script and mqtt climate in yaml (no so clean). And now I have found this library.

Could you give me an example on how to use it for climate entities in order to adapt my integration?

Thanks!

**Comment by ejohb** [comment]
You'll probably want to see the [dehumidifier example](https://github.com/fmtr/hct/blob/release/examples/electriq_cdx_pro_le.be) in the [examples directory](https://github.com/fmtr/hct/blob/release/examples).

I should probably note that I'm currently in the middle of replacing this library with [haco](https://github.com/fmtr/haco) (lets you do the same thing but using Python, via a Home Assistant addon) - but since its first release was today, be warned that it's very immature at the moment.

**Comment by soyxan** [comment]
Thank I will have a look to the examples.
As I am now in the position to "learn" how to use this hct library, do you think is better to start with haco?
Do you plan to stop the development of hct and move to haco?

**Comment by ejohb** [comment]
> Thank I will have a look to the examples. As I am now in the position to "learn" how to use this hct library, do you think is better to start with haco? Do you plan to stop the development of hct and move to haco?

Yeah I plan to set this project to read-only in the next couple of days.

If I were you I'd start with `haco`, but perhaps leave it a week or so until it's somewhat stable and some documentation exists. But if you want an idea of how do implement a climate control, have a look at the [test development config](https://github.com/fmtr/haco/blob/1c180f7fe6dce6ffccb7bd7165427eaceae22025/haco/configs/development.py#L29).

**Comment by soyxan** [comment]
Just for my understanding. With haco, there is no need to code in berry? Just in python?
Where is this python executed? In AppDaemon or something similar?
Can you give more details, I am very interested :)

**Comment by ejohb** [comment]
> Just for my understanding. With haco, there is no need to code in berry? Just in python? Where is this python executed? In AppDaemon or something similar? Can you give more details, I am very interested :)

Doc now up :) https://github.com/fmtr/haco/blob/release/README.md

**Comment by rlust** [comment]
Is this code still being developed? Have an error and wanted to see if support is available.

**Comment by ejohb** [comment]
Formally no. If it's a serious bug I won't fix it, but if it's a config issue I might be able to help. But please post a new issue either way.
